client/src/components/Button/useStyles.ts

```diff
@@ -73,7 +79,7 @@ const useStyles = makeStyles((theme) => ({
         color: theme.palette.common.white,
       },
       [theme.breakpoints.down('sm')]: {
-        color: '#f04040',
+        color: '#f04040 !important',
```

Comment:
to override the white color in small sizes screens,
It doesn't apply correctly without the !important rule,
I think it will override ALL previous styling rules for the color property on that element

Reply:
> color in the MUI theme palette
- done ✔️